doc/CHANGELOG.md

```diff
@@ -36,9 +36,13 @@ For F3D users:
  - Fixed an issue where focal point picking would generate error messages
  - Changed `--verbose` into a string based option, eg: `--verbose=quiet` or `--verbose=debug`. `--verbose` is still supported.
  - Changed `--no-render` behavior so that it doesn't impact verbosity anymore
- - Changed the default configuration file so that translucency support is enabled by default
- - Deprecated `--quiet`, use `--verbose=quiet` instead
- - Deprecated `--input`, use positional argument instead
+ - Deprecated `--quiet`, use `--verbose=quiet` instead.
+ - Reading EXR files is now multi-threaded and much faster.
+ - Translucency support is now enabled by default.
+ - A small margin is added between the model and the grid to avoid potential z-fighting.
+ - Add 3MF and X files support (part of `assimp` plugin)
+ - Fix potential crashes when reading `assimp` plugin file format
+ - Added ability to cycle through animations by pressing "W" key
```

Comment:
your changelog shoudl not be in 2.3.0 but in ongoing dev.

TBH you should just leave the changelog alone for the time being until just before we merge this :)